packages/create-cloudflare/src/common.ts

```diff
@@ -250,20 +252,47 @@ export const offerGit = async (ctx: PagesGeneratorContext) => {
 };
 
 export const gitCommit = async (ctx: PagesGeneratorContext) => {
-	if (!ctx.args.git) return;
+	if (!(await isGitInstalled()) || !(await isInsideGitRepo(ctx.project.path)))
+		return;
+
+	const commitMessage =
+		generateFrameworkCommitMessage(ctx) ??
+		"Initial commit (by Create-Cloudflare CLI)";
```

Comment:
to improve the readability of this code I suggest that you simplify this to 

```
const commitMessage =
		generateCommitMessage(ctx)
```

and this function you generate both fw and non-fw message.

Reply:
updated, thanks 🙂👍